src/components/legend/legend.jsx

```diff
@@ -1,15 +1,14 @@
 import React, { PropTypes } from 'react';
 import classNames from 'classnames';
 import { kebabCase } from 'lodash';
-// TODO: Move SCSS into JSS
-// import './legend.scss';
+import styleSheet from './legend-styles';
 
 function legendItem({ color, label, value, ...rest }) {
   return (
-    <li key={kebabCase(label)} {...rest} className="legend__item">
-      <span className="legend__item-indicator" style={{ backgroundColor: color }} />
-      <span className="legend__item-label">{ label }</span>
-      { value ? <span className="legend__item-value">{ value }</span> : null }
+    <li key={kebabCase(label)} {...rest} className="item">
```

Comment:
These classes should also use jss naming to avoid clashes. 
1. Flatten styles object (remove `& *` and bare classnames)
2. Use new classes like: `className={classes.item}` instead of bare classnames.